nowcasting_dataset/dataset/datasets.py

```diff
@@ -314,3 +364,91 @@ def worker_init_fn(worker_id):
         # The NowcastingDataset copy in this worker process.
         dataset_obj = worker_info.dataset
         dataset_obj.per_worker_init(worker_info.id)
+
+
+def subselect_data(
+    batch: example.Example,
+    required_keys: Union[Tuple[str], List[str]],
+    current_timestep_index: int,
+    history_minutes: int,
+    forecast_minutes: int,
+) -> example.Example:
+    """
+    Subselects the data temporally. This function selects all data within the range [history_minutes, forecast_minutes]
```

Comment:
This docstring looks great!

I'm probably being overly pedantic but this tiny tweak might be worth considering!  :)

```suggestion
    Subselects the data temporally. This function selects all data within the time range [t0 - history_minutes, t0 + forecast_minutes]
```